modules/installation-network-user-infra.adoc

```diff
@@ -64,6 +64,12 @@ ifeval::["{context}" == "installing-restricted-networks-gcp"]
 :gcp:
 :restricted:
 endif::[]
+ifeval::["{context}" == "installing-rhv-user-infra"]
```

Comment:
This assembly should be added to the top list for reference later:
```
// Module included in the following assemblies:
//
...
// * installing/installing-rhv-user-infra.adoc
```

Reply:
Fixed.